app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -164,6 +183,33 @@ const CORE_BOTTOM_TABS = [
   },
 ];
 
+const CORE_OTHER_NOTIFICATIONS_TAB = class extends UserMenuTab {
+  get id() {
+    return "other";
+  }
+
+  get icon() {
+    return "discourse-other-tab";
+  }
+
+  get panelComponent() {
+    return "user-menu/other-notifications-list";
+  }
+
+  get count() {
+    return this.site.unassignedNotificationTypes.reduce(
+      (sum, notificationType) => {
+        const key = this.site.notification_types?.[notificationType];
+        return (
+          sum +
+          (this.currentUser.get(`grouped_unread_notifications.${key}`) || 0)
```

Comment:
can we investigate using `this.getUnreadCountForType()` here? At some point, we'll no need the `get` workaround, so it'd be nice if the workaround was limited to just once place.